commons/src/main/java/org/infinispan/commons/configuration/attributes/AttributeSet.java

```diff
@@ -193,4 +233,7 @@ public void attributeChanged(Attribute<Object> attribute, Object oldValue) {
       // TODO
```

Comment:
Probably out of the scope - but could we throw an `UnsupportedOperationException` here? It's better than todo :)